instance/tests/models/test_openedx_instance.py

```diff
@@ -457,7 +459,7 @@ def test_get_load_balancer_configuration(self, mocks):
                 self.assertLogs("instance.models.instance", "ERROR"):
             appserver.server._public_ip = None
             appserver.server.save()
-            self.assertEqual(instance.get_load_balancer_configuration(), ([], []))
+            self.assertRaises(WrongStateException, instance.get_load_balancer_configuration)
```

Comment:
Please change the comment above this `with` block to reflect the new reality of what happens.